src/utils/constants.ts

```diff
@@ -68,3 +68,6 @@ export const GERALD_CONTRACT_ADDRESSES = [
   "0xe573BCA813c741229ffB2488F7856C6cAa841041",
   "0xBb0CC0fb3e0c06725c67167501f850B4900D6DB5",
 ];
+
+export const DOT_ASSET_ID = "42259045809535163221576417993425387648";
```

Comment:
Is that the same for all the networks ?

Reply:
It is for the relay chain token (dot, ksm, unit), so will rename appropriately.